### DISCUSSION TITLE
VT-x not available

### DISCUSSION THREAD

**Discussion by User2707** [magma/magma#15324] [Q&A]
There was an error while executing `VBoxManage`, a CLI used by Vagrant

for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "bc7c1fc0-3262-4f25-a48e-762d3268a038", "--type", "headless"]

Stderr: VBoxManage: error: VT-x is not available (VERR_VMX_NO_VMX)

VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole

I have enable KVM and VT-x in Virtualbox...Im still getting this error.
Any suggestions are welcome.Thank You. :)

**Comment by edaspb** [comment]
Hi! Probably VT-x is not enabled on BIOS or UEFI of the host machine.

**Comment by panyogesh** [comment]
@User2707 : We are planning to close the issue as its not magma specific.
